lib/Tie/Array/push.t

```diff
@@ -18,8 +21,9 @@ BEGIN {
  sub STORESIZE { $#{$_[0]} = $_[1]-1 }
 }
 
-tie @x,Basic;
-tie @get,Basic;
-tie @got,Basic;
-tie @tests,Basic;
+my (@x, @get, @got, @tests);
```

Comment:
I think these, and all of the similar Tie::Array tests, are intentionally affecting global variables for the following require, so these should be `our` declarations.

Reply:
Disturbingly if you remove the ties, the tests still pass. 😦

Reply:
I suspect the test isn't working right but that's out of scope here if we want to fix it.

Reply:
This isn't that surprising; the tests being required are already normal tests, this is just running them again with some tied variables.

Reply:
If we really wanted to make these tests robust, there would need to be something we could test after the require to ensure that the test actually operated on our tied variables.